### PR TITLE
W-12557250-sftp-release-2.0-kt

### PR DESCRIPTION
**Pull request by kevintroller**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: sftp/2.0/modules/ROOT/pages/sftp-connector-upgrade-guide.adoc [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+|1.6.0 |2.0.0
+|===
+
+== Changes in This Release
+
+This release contains the following changes:
+
+* The system property `mule.sftp.useHomeDirectory` was removed. SFTP Connector now uses the default home directory.
+* SFTP Connector no longer supports the `jcraft library`. The connector now uses `apache mina library`.
+* Changed the *List* operation name to *List and get*.
```
Comment (nicomz) [inline sftp/2.0/modules/ROOT/pages/sftp-connector-upgrade-guide.adoc:19]: Is this just the name? or the operation xml is also changed?

**Review by dmerlob — DISMISSED** [review]

File: sftp/2.0/modules/ROOT/pages/sftp-connector-upgrade-guide.adoc [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+|From Version | To Version
+|1.6.0 |2.0.0
+|===
+
+== Changes in This Release
+
+This release contains the following changes:
+
+* The system property `mule.sftp.useHomeDirectory` was removed. SFTP Connector now uses the default home directory.
+* SFTP Connector 2.0.0 no longer supports the `jcraft library`. The connector now uses `apache mina sshd library`.
```
Comment (sathya0) [inline sftp/2.0/modules/ROOT/pages/sftp-connector-upgrade-guide.adoc]:
```suggestion
* SFTP Connector 2.0.0 no longer supports the `jcraft library`. The connector now uses the `apache mina sshd library`.
```

File: sftp/2.0/modules/ROOT/pages/sftp-connector-upgrade-guide.adoc [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+* The *List* operation does not fetch the content of each file, instead, the payload of each list item contains the complete path to the found file or directory.
```
Comment (sathya0) [inline sftp/2.0/modules/ROOT/pages/sftp-connector-upgrade-guide.adoc]:
```suggestion
* The *List* operation no longer fetches the content of each file. Now, the payload of each list item contains the complete path to the found file or directory. 
```

File: sftp/2.0/modules/ROOT/pages/sftp-connector-upgrade-guide.adoc [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+== Changes in This Release
+
+This release contains the following changes:
+
+* The system property `mule.sftp.useHomeDirectory` was removed. SFTP Connector now uses the default home directory.
+* SFTP Connector 2.0.0 no longer supports the `jcraft library`. The connector now uses `apache mina sshd library`.
+* The *List* operation does not fetch the content of each file, instead, the payload of each list item contains the complete path to the found file or directory. 
+* The SFTP *Regular Size* file attribute is renamed to *Regular File* because its behavior is not related to size. The file attribute value can be a file (`true`) or a directory (`false`).
+* SFTP Connector 2.0.0 does not support ciphers `3des-ctr` and `arcfour`. See <<supported-ciphers,Supported Ciphers project>>.
```
Comment (sathya0) [inline sftp/2.0/modules/ROOT/pages/sftp-connector-upgrade-guide.adoc]:
```suggestion
* SFTP Connector 2.0.0 no longer supports ciphers `3des-ctr` and `arcfour`. For more information, refer to <<supported-ciphers,Supported Ciphers>>.
```

File: sftp/2.0/modules/ROOT/pages/sftp-connector-upgrade-guide.adoc [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+* The SFTP proxy protocol works only with HTTP and SOCKS5.
+* SFTP Connector no longer uses `javax.activation` to resolve the media types of the files. The connector now utilizes `Apache Tika`, which can result in different media file interpretations.
```
Comment (sathya0) [inline sftp/2.0/modules/ROOT/pages/sftp-connector-upgrade-guide.adoc]:
```suggestion
* SFTP Connector no longer uses `javax.activation` to resolve the media types of the files. The connector now uses `Apache Tika`, which can result in different media file interpretations.
```

**Review by sathya0 — DISMISSED** [review]

**Review by sathya0 — DISMISSED** [review]

**Review by sathya0 — DISMISSED** [review]

**Review by dmerlob — APPROVED** [review]

**Pull request merged**